messages/protos/messages.proto

```diff
@@ -591,6 +592,7 @@ message SubMessage {
     AskMuteVideoMessage askMuteVideoMessage = 28;
     ProximityPublicMessageToClientMessage proximityPublicMessageToClientMessage = 29;
     ProximityPrivateMessageToClientMessage proximityPrivateMessageToClientMessage = 30;
+    TypingProximityMessageToClientMessage typingProximityMessageToClientMessage = 31;
```

Comment:
We should definitely consider refactoring this by merging `ProximityPublicMessageToClientMessage` and `TypingProximityMessageToClientMessage` into a common message. This will remove a ton of redundant code.

Here is a proposal for the messages:

```
message PublicEvent {
  string spaceName = 1;
  SpaceEvent spaceEvent = 2;
}

message SpaceEvent {
  oneof event {
    SpaceMessage spaceMessage = 1;
    SpaceIsTyping spaceIsTyping = 2;
  }
}

message SpaceMessage {
  string message = 1;
}

message SpaceIsTyping {
  bool isTyping = 1;
}
```

This way, all the code that dispatches the messages in the pusher and the back can be coded only once.
And in the future, if we add additional messages, there will be no need to touch the pusher and the back